plugins/octant/cmd/antrea-traceflow-plugin/main.go

```diff
@@ -0,0 +1,401 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"regexp"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/vmware-tanzu/octant/pkg/navigation"
+	"github.com/vmware-tanzu/octant/pkg/plugin"
+	"github.com/vmware-tanzu/octant/pkg/plugin/service"
+	"github.com/vmware-tanzu/octant/pkg/view/component"
+	"github.com/vmware-tanzu/octant/pkg/view/flexlayout"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/clientcmd"
+
+	opsv1alpha1 "github.com/vmware-tanzu/antrea/pkg/apis/ops/v1alpha1"
+	clientset "github.com/vmware-tanzu/antrea/pkg/client/clientset/versioned"
+	"github.com/vmware-tanzu/antrea/pkg/graphviz"
+)
+
+var (
+	pluginName                           = "antreaTraceflowPlugin"
+	addTfAction                          = pluginName + "/addTf"
+	showGraphAction                      = pluginName + "/showGraphAction"
+	client          *clientset.Clientset = nil
+	kubeConfig                           = "KUBECONFIG"
+)
+
+const (
+	tfNameCol       = "Trace"
+	srcNamespaceCol = "Source Namespace"
+	srcPodCol       = "Source Pod"
+	srcPortCol      = "Source Port"
+	dstNamespaceCol = "Destination Namespace"
+	dstPodCol       = "Destination Pod"
+	dstPortCol      = "Destination Port"
+	protocolCol     = "Protocol"
+	crdCol          = "Detailed Information"
+	phaseCol        = "Phase"
+	ageCol          = "Age"
+
+	octantTraceflowCRDPath = "/cluster-overview/custom-resources/traceflows.ops.antrea.tanzu.vmware.com/v1alpha1/"
+
+	namespaceStrPattern = `[a-z0-9]([-a-z0-9]*[a-z0-9])?`
+	podStrPattern       = `[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*`
+
+	TIME_FORMAT_YYYYMMDD_HHMMSS = "20060102-150405"
+)
+
+// According to code in Antrea agent and controller, default protocol is ICMP if protocol is not inputted by users.
+const (
+	ICMPProtocol int32 = 1
+	TCPProtocol  int32 = 6
+	UDPProtocol  int32 = 17
+)
+
+var supportedProtocols = map[string]int32{
+	"ICMP": ICMPProtocol,
+	"TCP":  TCPProtocol,
+	"UDP":  UDPProtocol,
+}
+
+// This is antrea-traceflow-plugin.
+// The plugin needs to run with octant version v0.10.2 or later.
```

Comment:
Let's remove this line since we will run it with 0.13.1

Reply:
Removed.